server/lobby.js

```diff
@@ -39,6 +39,8 @@ class Lobby {
         this.lastUserBroadcast = moment();
 
         this.loadCardData();
+
+        setInterval(() => this.clearStalePendingGames(), 60000);
```

Comment:
I would typically break this out into its lower factors so it's slightly more clear what timeframe it is: ie 60 * 1000